deploy/manifests/prod/us-east-2/tenant/storetheindex/kustomization.yaml

```diff
@@ -2,10 +2,11 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: storetheindex
 resources:
+- assigner
 - instances
 - indexstar
 - heyfil
 images:
 - name: storetheindex
   newName: 407967248065.dkr.ecr.us-east-2.amazonaws.com/storetheindex/storetheindex # {"$imagepolicy": "storetheindex:storetheindex:name"}
-  newTag: 0.5.1 # {"$imagepolicy": "storetheindex:storetheindex:tag"}
+  newTag: 0.5.3 # {"$imagepolicy": "storetheindex:storetheindex:tag"}
```

Comment:
I recommend excluding this from the rollout and doing it instead via the auto-generated PR. 

Either way, once the new version is rolled out to indexer nodes, please remember to also update [kepa](https://github.com/ipni/storetheindex/blob/2c22c31f8fcf3551a2f07c2244334a7bf7b8b77b/deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/kepa/kustomization.yaml#L35)'s version since it is exculded from CD to reduce disruption on rollout.